.github/workflows/inshpect.yml

```diff
@@ -0,0 +1,48 @@
+name: Inshepct
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  doc:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Install tools
+        run: |
+          set -x
+          sudo apt-get update
+          sudo apt-get install -y fd-find
+          wget -q https://github.com/TomWright/dasel/releases/download/v2.7.0/dasel_linux_amd64 -O /tmp/dasel
```

Comment:
Please update this to also use `curl` and to check the checksum as with the other dependencies.

Reply:
This still needs the check for the checksum. Let me know if you need any help with adding that.